composer.json

```diff
@@ -25,12 +25,14 @@
         "symfony/yaml": "^5.4 || ^6.3 || ^7.0"
     },
     "require-dev": {
-        "doctrine/annotations": "^1.8.0 || ^2.0",
+        "doctrine/annotations": "^1.13.1 || ^2.0",
         "doctrine/data-fixtures": "^1.7",
+        "doctrine/dbal": "^3.7",
         "doctrine/doctrine-bundle": "^2.11",
         "doctrine/doctrine-fixtures-bundle": "^3.5.1 || ^4.0",
+        "doctrine/mongodb-odm": "^2.2",
```

Comment:
Same here about explicit dev dependencies.  The bundle says it supports all the way back to 2.0.0, but the CI build I referenced installs 2.1.2 and my PHP 8.2 localhost installed 2.2.  Either way it's a bit of an older version anyway so no big deal.